src/main/java/com/monitorjbl/xlsx/impl/StreamingSheetReader.java

```diff
@@ -280,7 +280,7 @@ String formattedContents() {
     switch(currentCell.getType()) {
       case "s":           //string stored in shared table
         if (!lastContents.isEmpty()) {
-            int idx = Integer.parseInt(lastContents);
+            int idx = Integer.parseInt(lastContents.trim());
```

Comment:
@bvn13 Cool, seems there are lots of issues around malformed SST references out in the wild. I saw some blank ones which is where the `if (!lastContents.isEmpty())` came in iirc.

Probably makes sense to change the if-statement from `if (!lastContents.isEmpty())` to `if (!lastContents.isEmpty() && !lastContents.trim().isEmpty())`, so if the value is just whitespace it won't throw a NumberFormatException.

Could you please add a unit test for the failing case? You could probably adapt `blank_sst_reference_doctored.xlsx` if you don't have a sample you can share. See `StreamingReaderTest.testShouldHandleBlankSSTReference()` for an example.